docs/navigation.md

```diff
@@ -0,0 +1,122 @@
+# Navigation panel
+
+This guide will walk you through the functions of each button in navigation panel. 
+
+# Navigation Button
+
+The first button in the navigation panel is the Navigation button. It is one of the default active buttons.
+
+Navigation button is divided into four parts. It provides the following information:
+
+- Servers
+- Operations
+- Messages
+- Schemas
+
+## Servers
+
+The Servers section displays the name and information of the server you are using to build your AsyncAPI.
+
+## Operations
+
+The Operations section lists all the operations that are a part of the AsyncAPI. It adds the perfix `PUB`  for publisher and `SUB` for subscriber to differentiate between both the operations.
+
+## Messages
+
+This Messages section displays the title of each message used to communicate to the API.
+
+## Schemas
+
+The Schemas displays the title of the schemas defined.
+
+
+# Editor
+
+The Editor section is where you edit the AsyncAPI code. As you edit, you will see the changes reflect dynamically in the HTML preview and Block Visualizer section.
+
+Click on the individual elements of the Navigation button to jump to those sections in HTML preview and Block Visualizer section.
+
+## Diagnostics Window
+
+The Editor also has resizeable Diagnostics window. It shows a quick count of number of errors (in red) and warnings (in yellow) by its title. The description of the warnings and errors with line number is displayed in a table.
+
+To the right side of the title bar there are three indicators of the Diagnostics window: 
+
+- Valid/Invalid - It is green if the syntax is green and red when invalid.
+- Autosave - It shows whether Autosave is on or off.
+- Type - Shows the language in which the API is written: `JSON` or `YAML`.
+
+Bwlow the title bar there is a control panel. You can the buttons to:
+
+- Hide and show erros
+- Hide and show warnings
+- Hide and show information messages
+- Hide and show hints
+
+There is also search bar to search that aids in finding issues by their names. You will see Settings icon to the right. It will take you to the governance of Diagnostic window.
```

Comment:
```suggestion
There is also search bar to search that aids in finding issues by their names. You will see Settings icon to the right. It will take you to the governance of Diagnostic's settings window.
```